web/src/beta/lib/core/Map/Layers/hooks.ts

```diff
@@ -561,6 +565,11 @@ export default function useHooks({
     setOverridenLayers(layers => layers.filter(l => !deleted.includes(l.id)));
   }, [atomMap, layers, layerMap, lazyLayerMap, setOverridenLayers, showLayer]);
 
+  useEffect(() => {
+    if (!requestingRenderMode || requestingRenderMode.current === -1) return;
```

Comment:
I think we should use the constant variable for this value. For example
```
const FORCE_REQUEST_RENDER  = -1;
const DISABLE_REQUEST_RENDER  = 0;
const ENABLE_REQUEST_RENDER  = 1;
```

Reply:
Updated!